packages/documentation/docs/controls/_date-input_styleguide.mdx

```diff
@@ -0,0 +1,69 @@
+
+![Date input overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3629-6200&t=ADQCetGKOEH1WG2r-4)
+1. Label
+2. Required field indicator
+3. Current value
+4. Calendar icon button
+5. Input field
+6. Date dropdown
+7. Month and year selection
+8. Weekdays
+9. Week numbers 
+10. Indicator for current day
+11. Indicator for selected day
+
+## Options
+
+- **Label**: See [form field](./forms/forms-field.md)
+- **Value**: See [form field](./forms/forms-field.md)
+- **Required**: See [form field](./forms/forms-field.md)
+- **Helper text**: See [form field](./forms/forms-field.md)
+- **Feedback text**: See [form field](./forms/forms-field.md)
+- **Show text as tooltip**: See [form field](./forms/forms-field.md)
+- **Placeholder**: See [form field](./forms/forms-field.md). We typically use a placeholder to show an example date format to assist users when the field is empty.
+- **Error message**: Feedback text when date is not parsable. We typically use this to inform users that the entered date format is incorrect and guide them to enter a valid date.
+- **Format**: Specify the date format, default ‘yyyy/LL/dd’. We typically use this to ensure that dates are entered in a consistent and recognizable format, leveraging the Luxon JavaScript library for parsing and formatting.
+
+## Behavior in context
+
+- **Interaction**:
+  - Click or focus opens date picker.
+  - Use mouse or keyboard arrows to navigate to desired date.
+  - Selecting a date in date picker with mouse click or enter closes the date picker.
+  - Typing a date into input field with valid format closes the date picker.
+  - Escape key closes the date picker.
+- **Validation**:
+  - Use feedback text for validation types valid, info, warning and invalid.
+  - Invalid feedback automatically provided if entered date in not parsable.
+  - Refer to the [validation](./forms/forms-validation.md) chapter for detailed guidelines.
+- **Overflow**: The input field should be wide enough to display the full date without truncation.
+- **Alignment**: Date inputs are always aligned to the left.
+
+## States
+
+Date input has five states: Default, hover, disabled, read-only and focused.
+
+![Date input states](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3989-2545&t=ADQCetGKOEH1WG2r-4)
+
+
+## Dos and Don’ts
+
+- Do use consistent date formats: Ensure the date format is uniform throughout the application to avoid confusion
+- Do use separate inputs for date ranges: Provide distinct input fields for the start and end dates to make it easier for users to enter and understand date ranges
+- Do provide clear instructions: Clearly indicate the required date format to users. For example, “Please enter the date in yyyy-MM-dd format”
```

Comment:
```suggestion
- Do provide clear instructions: Clearly indicate the required date format to users. For example, “Enter the date in yyyy-MM-dd format”
```

In the writing guideline, we recommend to not use _please_ since it lengthens the sentence 😉